### ISSUE TITLE
Mech Island - Junkyard Tinkering

### ISSUE THREAD

**Issue by vandiel01** [ThingEngineering/wowthing-collector#27]
I was cleaning up my toon on ATT and naturally ATT would say "learn this/that again" so I had to go to places and re-learn some, no biggie anyways, ATT tells me I'm missing Recipes from Junkyard Tinkering in Mech Island, was like no, I already know it on Main, went there, open Junkyard Tinkering and got this error:

```
2x ...dOns/WoWthing_Collector/Modules/ProfessionOrders.lua:41: bad argument #1 to 'IsNearProfessionSpellFocus' (Usage: local nearFocus = C_TradeSkillUI.IsNearProfessionSpellFocus(profession))
[string "=[C]"]: in function `IsNearProfessionSpellFocus'
[string "@WoWthing_Collector/Modules/ProfessionOrders.lua"]:41: in function `RequestOrders'
[string "@WoWthing_Collector/Modules/ProfessionOrders.lua"]:32: in function `?'
[string "@AllTheThings/lib/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...ings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@AllTheThings/lib/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...ings/lib/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@AllTheThings/lib/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@BagSync/libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <...ceBagSync/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
(*temporary) = nil
```

**Comment by madcowfred** [comment]
This should be fixed with v11.0.2.30

**Comment by vandiel01** [comment]
No error message this time, thanks!
